src/core/credrank/markovProcessGraph.js

```diff
@@ -958,6 +958,7 @@ function virtualizedMarkovEdge(
         personalAttributionAddress.fromParticipantId,
         personalAttributionAddress.toParticipantId
       ) || 0;
+    if (proportionValue === 0) return null;
```

Comment:
Is this needed? is this purely for robustness, since it isn't really reachable due to address filtering?

Reply:
This _is_ needed though, since it will yield 0-weighted edges when called from `mpg.edge()`. Do we want virtualized edges with 0-weight to return from this query?

Reply:
I mean that mpg.edge() call should never happen because where would someone get the address except by some sort of forceful construction? But it's fine, makes it more robust.